docs/db_structure.md

```diff
@@ -26,6 +26,7 @@ CREATE TABLE report (
     report          VARCHAR NOT NULL,
     reported_at     TIMESTAMP,
     last_checked_at TIMESTAMP,
+    kafka_offset BIGINT  NOT NULL DEFAULT 0,
```

Comment:
I guess eventually it will be removed from the code.

Reply:
well, not necessarily, it can be used for debugging